cloudsync/sync/manager.py

```diff
@@ -1388,6 +1394,25 @@ def embrace_change(self, sync, changed, synced):  # pylint: disable=too-many-ret
         log.debug("nothing changed %s", sync)
         return FINISHED
 
+    def handle_changed_is_missing(self, sync, changed, synced):     # pylint: disable=no-self-use
+        if (sync[synced].exists == EXISTS and sync.paths_match(changed) and
+            sync[changed].hash == sync[changed].sync_hash):
+            #TODO: and not sync[synced].changed -- another condition that was in Mike's original change
```

Comment:
@earthman1 - looking to get some clarification in the conditions here - my understanding is that these ensure the file didn't actually change (in which case we want to punt and let a future iteration handle the change). But, I had to remove the .changed check to get tests to pass...

Reply:
The tests are probably the best guide on what the correct condition is here. I don't see a problem with removing the check on the changed flag